Terminal.Gui/View/ViewDrawing.cs

```diff
@@ -365,7 +375,7 @@ public virtual Attribute GetNormalColor ()
     {
         ColorScheme cs = ColorScheme;
 
-        if (ColorScheme is null)
+        if (cs is null)
```

Comment:
Any block like this:
```csharp
if( cs is null )
{
  cs = new(); // Any assignment - not just new()
}
```

should be this:

```csharp
cs ??= new();
```

[Null coalescing assignment](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-8.0/null-coalescing-assignment).

It is exactly the same, without the extra cyclomatic complexity and, if the compiler can reason about what happens above, it may be able to optimize away the check in some cases.